sekizai/templatetags/sekizai_tags.py

```diff
@@ -41,6 +41,18 @@ def import_processor(import_path):
     return getattr(module, object_name)
 
 
+def import_mapped_processors():
+    preprocessors, postprocessors = {}, {}
+    sekizai_processors = getattr(settings, 'SEKIZAI_PROCESSORS', {})
+    for name, processor in sekizai_processors.get('pre', {}).items():
+        preprocessors[name] = import_processor(processor)
+    for name, processor in sekizai_processors.get('post', {}).items():
+        postprocessors[name] = import_processor(processor)
+    return preprocessors, postprocessors
+
+_mapped_preprocessors, _mapped_postprocessors = import_mapped_processors()
```

Comment:
not a fan of these being globals that get populated at import time. this makes testing very difficult and is generally considered sub-optimal. I think these should be populated somewhere like ``RenderBlock.__init__`` (not 100% sure where to do it best, but somewhere other than at import time).